pkg/relay.go

```diff
@@ -405,15 +408,18 @@ func SubmitBlockRequestToSignedBuilderBid(req *types.BuilderSubmitBlockRequest,
 
 // SubmitBlock Accepts block from trusted builder and stores
 func (rs *DefaultRelay) SubmitBlock(ctx context.Context, submitBlockRequest *types.BuilderSubmitBlockRequest, state State) error {
-	logger := rs.Log().WithField("method", "SubmitBlock")
 	timeStart := time.Now()
 
-	logger.With(log.F{
-		"processingTimeMs": time.Since(timeStart).Milliseconds(),
-		"slot":             submitBlockRequest.Message.Slot,
-		"blockHash":        submitBlockRequest.ExecutionPayload.BlockHash,
-		"proposer":         submitBlockRequest.Message.ProposerPubkey,
-	}).Trace("block submission requested")
+	logger := rs.Log().With(log.F{
+		"method":    "SubmitBlock",
+		"builder":   submitBlockRequest.Message.BuilderPubkey,
```

Comment:
This is big enough that I think a `Loggable` wrapper type for `types.BuilderSubmitBlockRequest` is warranted.